Source/WebSocket.swift

```diff
@@ -1035,7 +1037,7 @@ private class InnerWebSocket: Hashable {
 			security = .none
 		}
 
-        var path = CFURLCopyPath(req.url! as CFURL?) as String
+		var path = CFURLCopyPath(req.url! as CFURL) as String
```

Comment:
extra spaces on the left

Reply:
Now it's on the same level 
<img width="838" alt="Screen Shot 2019-12-11 at 17 58 03" src="https://user-images.githubusercontent.com/32259711/70637426-e17ffc00-1c3f-11ea-8449-5b28428e9001.png">